web/src/dropdown_widget.ts

```diff
@@ -145,7 +145,7 @@ export class DropdownWidget {
             `${this.widget_selector}, ${this.widget_wrapper_id}`,
             (e) => {
                 if (e.key === "Enter") {
-                    $(this.widget_selector).trigger("click");
+                    $(this.widget_selector)[0]?.click();
                     e.stopPropagation();
```

Comment:
> Keyboard navigation doesn't open saved snippets from the menu -- pressing Enter with the option selected just makes the focus ring disapper.

I had to make this change to resolve this bug. Can it have any unexpected behaviour - using `click()` instead of `trigger("click")`?

Reply:
This may be a pattern we want to avoid, because it makes migrating away from jQuery harder, but we do use it in the codebase, and it's not a huge deal.

Oddly, I can confirm it is required to make this work. I really don't understand how the difference between these two ways of triggering a click could possibly be important. 

Maybe I can merge this, but @amanagr can investigate and post in #frontend his findings.